### ISSUE TITLE
Table view size

### ISSUE THREAD

**Issue by CursedHades** [RafalSwat/Shared_List#2]
https://github.com/RafalSwat/Shared_List/blob/d7518f47fcce027d1e983e7385a7b601d280fc3d/Shared_List/Controller/ListViewController.swift#L40-L53

Hardcoded array with items is ok for testing, but the way you did that, can hit you in the face in the future. The problem is the array is local for one of the functions, so another one has harcoded magic value of 3. At some point you will move this array to a class attribute and fill it with data fetched from database. It is expected that NumberOfRows uses array.size, so you might forget to change it and bum! you have a bug and spending time debugging database connection (because setting UI is so easy, so it is fine for sure).

**Comment by RafalSwat** [comment]
Yeah yeah, I have this on my mind. Actually, Angela fix this "bug" later on. It was only for testing. But you are right, it could be easy forgotten
